src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -990,8 +990,10 @@ InferenceProfiler::MergePerfStatusReports(
   experiment_perf_status.client_stats.delayed_request_count = 0;
   experiment_perf_status.client_stats.duration_ns = 0;
   experiment_perf_status.client_stats.avg_latency_ns = 0;
+  experiment_perf_status.client_stats.avg_response_latency_ns = 0;
```

Comment:
I don't love the use of latency vs response_latency variable names everywhere. What is the difference? Can the variables be more clearly named?

Reply:
ok then the names should reflect that ;)